examples/check_creations.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+path=`pwd`
+some=${path##*/}
+cd keystore
+
+if [ $some = "alice" ];
+then
+    ## alice has all key types (11) thus we just count the number of files in keystore dir
+    count=$(ls | wc -l)
+    if [ $count = 11 ]; then
+        exit 0
+    fi
+elif [ $some = "bob" ];
+then
+    result=`ls $(echo -n aura | hexdump -v -e '/1 "%02X"')*`
+    count=$(echo -e "$result" | wc -l)
+    if [ $count = 1 ]; then
+        exit 0
+    fi
+elif [ $some = "collator01" ];
+then
+    result=`ls $(echo -n gran | hexdump -v -e '/1 "%02X"')*`
+    count=$(echo -e "$result" | wc -l)
+    exit 0
+fi
+
+exit 1
+echo "ERRORED"
```

Comment:
this line will never be printed since we are exit before.

Reply:
Also, I think this test will only work using the `native` provider. Since the path of the keystore inside the `pods` is different.

Reply:
Can you elaborate a this bit more? All the paths I use inside are relevant (initiated by `pwd`). I will continue testing on podman and k8s but I think if we have these inconsistencies between providers maybe we should create a "path mapping" between supported providers in order to be able to be consistent

Reply:
I think what @pepoviola means is that the ```exit 1``` is like a "return" so the rest of the script after it (the ```echo "ERRORED"``` in that case) won't be executed

Reply:
Yes, we follow the same directories structure in all the providers in the `nodes`

```
/data --> we use as base-path for running the node, here you will find the db, keystore, etc.
/relay-data --> Same as `/data` but only used in collators as base-path for the full-node of the relaychain.
/cfg --> we store the config and extra scripts, here you will find the chain-spec, and all the scripts we upload as part of the test-runner.
```

In the case of `native` provider the `tmpDir` acts as the `/` of the `node` fs.